### ISSUE TITLE
Varia: Primary background color set on classic block toolbar

### ISSUE THREAD

**Issue by brookewp** [Automattic/themes#3319]
## Steps to replicate

1. Switch theme to a Varia theme (I tested Shawburn, Stow, Hever)
2. Add Classic block
3. See background color on toolbar

<img width="798" alt="Screen Shot 2021-02-21 at 11 04 57 AM" src="https://user-images.githubusercontent.com/35543432/108635608-15a29f80-7435-11eb-8bf3-b554bae48474.png">

## Result

The primary background color is on the toolbar buttons. This makes them very difficult to read/not very accessible. 

## Expected

For the default styles of the toolbar to show instead.

* User report: 3760468-zen
* Follow-up ticket: same as above

**Comment by kathrynwp** [comment]
I encountered a Quick Start customer today with a similar issue, but on **every** block, not just the Classic one. They are using Stow.

![Zoom_Meeting](https://user-images.githubusercontent.com/7662449/108756878-1b70b180-7517-11eb-81ba-01d85a9058b4.jpg)

3764195-zen

User said the issue just started recently. Happens in Chrome & Firefox, user said it does not happen in Safari.

**Comment by kathrynwp** [comment]
Note that the buttons in the editor interface also has rounded edges in the screenshots above. They should have straight edges, like this:

![Edit_Page_‹_STOW_—_WordPress](https://user-images.githubusercontent.com/7662449/108757368-c08b8a00-7517-11eb-8fa7-73b33436fb2a.jpg)
